config.inc.php.template

```diff
@@ -18,9 +18,15 @@ define('OBJECTCLASS', 'inetOrgPerson');
 define('PAGETITLE', 'example.com User Admin');
 
 define('MAIL_SENDER', 'webteam@example.com');
-define('MAIL_SUBJECT', 'Your account for example.com');
-const MAIL_TEMPLATES = array('Default' => 'mail-templates/mail-newuser.tpl',
-  'Umstellung' => 'mail-templates/mail-newuser.umstellung.tpl');
+const MAIL_TEMPLATES = array(
+  'addUser' = array(
```

Comment:
Do we really need to distinguish two types of templates or should we always offer all templates (the variables in the template are the same)